File: test/fixtures/csharp/get_with_env_var.cs
using System;
using System.Net.Http;
using System.Net.Http.Headers;

HttpClient client = new HttpClient();

HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Get, "http://localhost:28139/v2/images?type=distribution");

request.Headers.Add("Authorization", "Bearer " + Environment.GetEnvironmentVariable("DO_API_TOKEN"));

request.Content = new StringContent("");
request.Content.Headers.ContentType = new MediaTypeHeaderValue("application/json");

HttpResponseMessage response = await client.SendAsync(request);
response.EnsureSuccessStatusCode();
string responseBody = await response.Content.ReadAsStringAsync();

File: test/fixtures/swift/get_with_env_var.swift
import Foundation

let url = URL(string: "http://localhost:28139/v2/images?type=distribution")!
let headers = [
    "Content-Type": "application/json",
    "Authorization": "Bearer " + (ProcessInfo.processInfo.environment["DO_API_TOKEN"] ?? "")
]

var request = URLRequest(url: url)
request.allHTTPHeaderFields = headers

let task = URLSession.shared.dataTask(with: request) { (data, response, error) in
    if let error = error {
        print(error)
    } else if let data = data {
        let str = String(data: data, encoding: .utf8)
        print(str ?? "")
    }
}

task.resume()
